ultralist/date_filter.go

```diff
@@ -188,9 +188,11 @@ func (f *DateFilter) filterOverdue(pivot time.Time) []*Todo {
 	pivotDate := pivot.Format("2006-01-02")
 
 	for _, todo := range f.Todos {
-		dueTime, _ := time.ParseInLocation("2006-01-02", todo.Due, f.Location)
-		if dueTime.Before(pivot) && pivotDate != todo.Due {
-			ret = append(ret, todo)
+		if todo.Due != "" {
```

Comment:
The overdue filter was always displaying todos without due date.

Reply:
could you push this into a separate PR?  I agree that todos without a due date shouldn't be part of the `overdue` filter.

Reply:
See #169 and #170.